notebook/config_manager.py

```diff
@@ -0,0 +1,104 @@
+"""Manager to read and modify config data in JSON files.
+"""
+# Copyright (c) IPython Development Team.
```

Comment:
Since you are porting this from traitlets, it shows some oldies... let's use the updated header.